tests/conftest.py

```diff
@@ -4,13 +4,14 @@
 from pathlib import Path
 
 import pytest
+from pytmc.pragmas import normalize_io
+
```

Comment:
Empty line in the wrong place?

Reply:
There seem to be a bunch of instances of this scattered

Reply:
Not sure why `isort` did that...

Reply:
It seems to think pytmc is a third party package while ophyd is first party?
This can be fixed by adding a couple lines to setup.cfg:
https://github.com/timothycrosley/isort#configuring-isort

Reply:
I traced it back to an issue in isort which is fixed in an as-of-yet unreleased version. Changes are now in c8574ae , which is now just separating the `import` and `from` blocks.